lhotse/supervision.py

```diff
@@ -215,6 +215,32 @@ def perturb_speed(
             } if self.alignment else None
         )
 
+    def perturb_tempo(
+            self,
+            factor: float,
+            sampling_rate: int,
+            affix_id: bool = True
+    ) -> 'SupervisionSegment':
+        """
+        Return a ``SupervisionSegment`` that has time boundaries matching the
+        recording/cut perturbed with the same factor.
+
+        :param factor: The tempo will be adjusted this many times (e.g. factor=1.1 means 1.1x faster).
+        :param sampling_rate: The sampling rate is necessary to accurately perturb the start
+            and duration (going through the sample counts).
+        :param affix_id: When true, we will modify the ``id`` and ``recording_id`` fields
+            by affixing it with "_tp{factor}".
+        :return: a modified copy of the current ``Recording``.
+        """
+
+        # speed and tempo perturbation have the same effect on supervisions
+        perturbed = self.perturb_speed(factor, sampling_rate, affix_id)
```

Comment:
You probably want to prevent affixing ID in perturb_speed like this:

```suggestion
        perturbed = self.perturb_speed(factor, sampling_rate, affix_id=False)
```